src/response.rs

```diff
@@ -151,6 +151,7 @@ impl ResponseHeaders {
         self.write_allow_header(buf)?;
         self.write_deprecation_header(buf)?;
 
+
```

Comment:
Unnecessary change?

Reply:
ya, I write the content_length to response header no matter content_length is 0, but after I check the http spec carefully, I found if the status code is 1xx or 204, we must not set the content length header, so I revert the change. However, we may need to consider the case when http status code is 200 and content_length is zero, in this case content length should be set to the response header